tests.py

```diff
@@ -228,87 +237,82 @@ def test_random_ind_2():
 exported_pipeline.fit(training_features, training_classes)
 results = exported_pipeline.predict(testing_features)
 """
+
     assert expected_code == export_pipeline(pipeline, tpot_obj.operators, tpot_obj._pset)
 
-def test_score():
-    """Assert that the TPOT score function raises a RuntimeError when no optimized pipeline exists"""
 
+def test_score():
+    """Assert that the TPOT score function raises a RuntimeError when no optimized pipeline exists."""
     tpot_obj = TPOTClassifier()
-
-    try:
-        tpot_obj.score(testing_features, testing_classes)
-        assert False  # Should be unreachable
-    except RuntimeError:
-        pass
+    assert_raises(RuntimeError, tpot_obj.score, testing_features, testing_classes)
 
 
 def test_score_2():
-    """Assert that the TPOTClassifier score function outputs a known score for a fix pipeline"""
-
+    """Assert that the TPOTClassifier score function outputs a known score for a fix pipeline."""
     tpot_obj = TPOTClassifier()
     known_score = 0.977777777778  # Assumes use of the TPOT balanced_accuracy function
```

Comment:
May need to delete or update this comment since the default scoring function is `accuracy` instead of `balanced_accuracy`